processor/cumulativetodeltaprocessor/processor.go

```diff
@@ -69,7 +69,7 @@ func (ctdp *cumulativeToDeltaProcessor) processMetrics(_ context.Context, md pda
 							labelMap := make(map[string]string)
 
 							fromDataPoint.Attributes().Range(func(k string, v pdata.AttributeValue) bool {
-								labelMap[k] = v.StringVal()
+								labelMap[k] = pdata.AttributeValueToString(v)
```

Comment:
Do you use this only for the map? Do you reconstruct the labels from this?

Reply:
we are using this stringMap to generate a key for mapping. Though we are creating new attrValues from this map but not considering to change or depend on the type of these attributes. At least that I can say looking into the code here.
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/internal/aws/metrics/metric_calculator.go#L94-L105